src/base/application.cpp

```diff
@@ -421,36 +422,54 @@ std::string Application::findInputFile(const std::string& name)
             home = getenv("USERPROFILE"); // Windows systems
         }
         if (home) {
-            return home + name.substr(1, npos);
-        }
-    }
-
-    if (islash == string::npos && ibslash == string::npos) {
-        size_t nd = dirs.size();
-        for (size_t i = 0; i < nd; i++) {
-            string inname = dirs[i] + "/" + name;
-            std::ifstream fin(inname);
+            string full_name = home + name.substr(1, npos);
+            std::ifstream fin(full_name);
             if (fin) {
-                return inname;
+                return full_name;
+            } else {
+                throw CanteraError("Application::findInputFile",
+                                "Input file '{}' not found", name);
```

Comment:
I think the indentation here should line up
```suggestion
                                   "Input file '{}' not found", name);
```